apps/web/vibes/soul/examples/sections/products-list-section/electric-loading.tsx

```diff
@@ -0,0 +1,44 @@
+import { getBreadcrumbs, getFilters, getProducts, getSortOptions } from '@/vibes/soul/data'
+import { ProductsListSection } from '@/vibes/soul/sections/products-list-section'
+import { Filter, Option, Product } from '@/vibes/soul/types'
+
+export default function Preview() {
+  const products = new Promise<Product[]>(res => {
+    setTimeout(() => {
+      res(getProducts('electric'))
```

Comment:
I don't think this will handle errors, but for the purposes of demoing loading state, I think this is ok? I imagine will have a separate example for showing errors?

Reply:
Will do a pass for errors when ready.